File: app/src/main/java/ru/demo/messenger/network/response/ChatResponse.java
package ru.demo.messenger.network.response;

import com.google.gson.annotations.SerializedName;

import ru.demo.messenger.data.chat.ChatModel;
import ru.demo.messenger.network.response.base.BaseResponse;

public class ChatResponse extends BaseResponse {
    @SerializedName("private_message")
    public ChatModel chat;
}

File: app/src/main/java/ru/demo/data/message/ChatResponse.kt
package ru.demo.data.message

import com.google.gson.annotations.SerializedName
import ru.demo.data.network.BaseResponse

import ru.demo.messenger.data.chat.ChatModel

class ChatResponse(
        @SerializedName("private_message")
        var chat: ChatModel
) : BaseResponse()
